src/server/documents.ts

```
@@ -98,6 +114,20 @@ export async function uploadDocument(
 
         uniqueKey = `${orgPathSegment}/projects/${po.project.projectNumber}/purchase-orders/${monthFolder}/${poFolder}/${file.name}`;
       }
+    } else if (linkedEntity?.extensionId) {
```

Biome (2.1.2)

[error] This branch can never execute. Its condition is a duplicate or covered by previous conditions in the if-else-if chain.



(lint/suspicious/noDuplicateElseIf)